bin/taste-tester

```diff
@@ -368,6 +375,24 @@ MODES:
     ) do
       options[:skip_post_test_hook] = true
     end
+
+    opts.on(
+      '--use-custom-changeset-hook', 'Find a custom changeset. Works on impact.'
```

Comment:
This... seems all sorts of scary to me. Let people pass in a start/end, but bypassing the changset calculation doesn't seem like a good idea to me. That's the whole point of TT/BM/GD. What is it you want to do here, we can just extend BM almost certainly...

Reply:
The idea here was to allow users to define their own logic for calculating what has been changed.  However, I think you are correct that this is already covered by the Config.repo and a start/end ref.  I have removed the hook and added an additional config option.